### PR TITLE
Update descriptions for "template" and "parameters". 

### PR DESCRIPTION
**Pull request by TianoMS**
And Remove "gets or sets" from all descriptions.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@TianoMS__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: src/ResourceManagement/Resource/Microsoft.Azure.Management.ResourceManager/Generated/Models/AliasPathType.cs [hunk 1]
```diff
@@ -35,13 +35,13 @@ public AliasPathType(string path = default(string), IList<string> apiVersions =
         }
 
         /// <summary>
-        /// Gets or sets the path.
+        /// The path.
```
Comment (gandhiniraj) [inline src/ResourceManagement/Resource/Microsoft.Azure.Management.ResourceManager/Generated/Models/AliasPathType.cs]: nit: add more detail about the path

**Comment by markcowl** [comment]
@TianoMS One comment from your team, can you update, please?

File: src/ResourceManagement/Resource/Microsoft.Azure.Management.ResourceManager/Generated/Models/AliasPathType.cs (hunk 1 above)
Comment (TianoMS) [inline src/ResourceManagement/Resource/Microsoft.Azure.Management.ResourceManager/Generated/Models/AliasPathType.cs]: Fixed.

**Comment by markcowl** [comment]
@TianoMS, you will need to update the version (project.json and AssemblyInfo.cs) to publish the new package

**Comment by TianoMS** [comment]
@markcowl I've already updated the version to 1.1.4-preview in another PR.

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by hovsepm** [comment]
@azuresdkci retest this please

**Pull request merged**